internal/nexodus/nexodus.go

```diff
@@ -398,8 +400,17 @@ func (ax *Nexodus) Start(ctx context.Context, wg *sync.WaitGroup) error {
 		}
 	}
 
-	if err := ax.Reconcile(true); err != nil {
-		return fmt.Errorf("initial reconcile failed: %w", err)
+	err = util.RetryOperation(ctx, 15*time.Second, 3, func() error {
+		if err := ax.Reconcile(true); err != nil {
+			if err != nil {
+				ax.logger.Debugf("initial reconciliation failed - retrying: %v", err)
```

Comment:
I think we should dump this log as a Warn?

Reply:
Until we get it stable it would be a lot of warns. I can warn on all of the retries if we want. These are as much to get information for getting stable.

Reply:
In general (at-least in production) if the retry is failing, we should dump it as a warn to ensure we capture the max details. I think for dev environment we can disable the stack track. I think we should configure `logCfg.DisableStacktrace = true` for zap.NewDevelopment() and remove it for zap.NewProductionConfig()? I think it should solve this issue ?

Reply:
WFM, added. Also re-did the unit test as it failed enough times on windows to make me nervous ¯\\_(ツ)_/¯